renku/ui/service/serializers/datasets.py

```diff
@@ -181,6 +181,7 @@ class DatasetImportRequest(AsyncSchema, LocalRepositorySchema, RemoteRepositoryS
     dataset_uri = fields.String(required=True)
     name = fields.String(description="Optional dataset name.")
     extract = fields.Boolean()
+    tag = fields.String(description="Dataset version to import.")
```

Comment:
We need to bump the API (minor) version when adding fields, so a client can know whether this feature is supported or not.

See https://github.com/SwissDataScienceCenter/renku-python/blob/develop/renku/ui/service/views/api_versions.py#L62 and add the version here: https://github.com/SwissDataScienceCenter/renku-python/blob/develop/renku/ui/service/views/datasets.py#L191 (And all other endpoints that have 1.1, which is annoying)